### PR TITLE
[1.3] Bump go version to 1.22.8

### PR DESCRIPTION
**Pull request by PyAgni**
https://github.com/etcd-io/etcd/issues/18665

### CODE REVIEW

File: .github/workflows/tests.yaml
Comment (ivanvc) [inline .github/workflows/tests.yaml:1]: Minor nit: Can you split the commit into two, one for the golangcli-lint version update and another for Golang's?

Same applies to #836

Comment (ahrtr) [reply .github/workflows/tests.yaml:1]: > Can you split the commit into two, one for the golangcli-lint version update and another for Golang's?

Yes, I agree. But I won't insist on it for such minor changes, and we won't update the changelog for either change.

**Review by ivanvc — APPROVED** [review]
Per https://github.com/etcd-io/bbolt/pull/837#discussion_r1785859261, this LGTM. Thanks, Agni.

**Comment by ivanvc** [comment]
Quick question, @ahrtr: did you manually approve the workflows for the pull request? I see it skipped the `ok-to-test` label, but I'm unsure why the tests ran.

**Comment by ahrtr** [comment]
> Quick question, @ahrtr: did you manually approve the workflows for the pull request?

Yes, maintainers need to manually trigger workflow for first-time contributors. Please feel free to introduce ok-to-test mechanism to bbolt if you have time. thx

**Review by ahrtr — APPROVED** [review]

**Comment by ivanvc** [comment]
> Yes, maintainers need to manually trigger workflow for first-time contributors. Please feel free to introduce ok-to-test mechanism to bbolt if you have time. thx

I forgot to follow up on this comment. We have it in place (https://github.com/etcd-io/bbolt/pull/638), but I'm not sure why every new PR in the etcd repo has the needs-ok-to-test and not here.

**Comment by ahrtr** [comment]
> I forgot to follow up on this comment. We have it in place (#638), but I'm not sure why every new PR in the etcd repo has the needs-ok-to-test and not here.

It seems that you did not get release-1.3 included in the list?

https://github.com/etcd-io/bbolt/blob/12280e06d5de2160374dafde6dbc194a1e7db9a3/.github/workflows/gh-workflow-approve.yaml#L9-L10

**Comment by ivanvc** [comment]
> It seems that you did not get release-1.3 included in the list?

Good catch. I'll open a PR and move the conversation there.

**Pull request merged**